tox.ini

```diff
@@ -43,10 +43,10 @@ deps =
     flake8-debugger
     flake8-deprecated
     flake8-todo
-    flake8-isort
     mccabe
     flake8-blind-except
     flake8-commas
 commands =
-    - isort --check-only --recursive {toxinidir}/src setup.py
+    isort --check-only --diff {toxinidir}/src setup.py
     flake8 --doctests src setup.py {posargs}
+    isort --version
```

Comment:
Is this needed?

Reply:
I overlooked this - I needed this for debugging some "special" behaviour of isort.

Removed it.